build-ceph-rpm.sh

```diff
@@ -54,7 +55,7 @@ echo --START-IGNORE-WARNINGS
 # we only need to use autogen here if we need a dist tarball
 if [ -x make-dist ]; then
     echo --STOP-IGNORE-WARNINGS
-    ./make-dist
+    ./make-dist $RPM_VER
```

Comment:
got it. but we will have the identical VERSION and RPM_RELEASE in the `ceph.spec`. currently they will both be "11.0.0". is this expected?